pybamm/models/submodels/interface/lithium_ion.py

```diff
@@ -0,0 +1,97 @@
+#
+# Lithium-ion interface classes
+#
+import pybamm
+from .base_interface import BaseInterface
+from . import kinetics
+
+
+class BaseModel(BaseInterface, pybamm.lithium_ion.BaseModel):
+    """
+    Base lead-acid interface class
+
+    Parameters
+    ----------
+    param :
+        model parameters
+    domain : str
+        The domain to implement the model, either: 'Negative' or 'Positive'.
+
+
+    **Extends:** :class:`pybamm.interface.BaseInterface`
+    and :class:`pybamm.lithium_ion.BaseModel`
+    """
+
+    def __init__(self, param, domain):
+        super().__init__(param, domain)
+
+    def _get_exchange_current_density(self, variables):
+        """
+        A private function to obtain the exchange current density for a lead acid
+        deposition reaction.
+
+        Parameters
+        ----------
+        variables: dict
+        `   The variables in the full model.
+
+        Returns
+        -------
+        j0 : :class: `pybamm.Symbol`
+            The exchange current density.
+        """
+        c_s_surf = variables[self.domain + " particle surface concentration"]
+        c_e = variables[self.domain + " electrolyte concentration"]
+
+        if self.domain == "Negative":
+            prefactor = 1 / self.param.C_r_n
+
+        elif self.domain == "Positive":
+            prefactor = self.param.gamma_p / self.param.C_r_p
+
+        j0 = prefactor * (
+            c_e ** (1 / 2) * c_s_surf ** (1 / 2) * (1 - c_s_surf) ** (1 / 2)
+        )
+
+        return j0
+
+    def _get_standard_ocp_variables(self, variables):
```

Comment:
Ah ok I see the issue. i agree that the order of the submodels shouldn't matter. But having `ocp` as an option was pretty jarring. Could the entropic change be set in `thermal._get_standard_coupled_variables` instead? Otherwise I'll just change this back